### PR TITLE
docs: add troubleshooting

### PR DESCRIPTION
**Pull request by anfibiacreativa**
WIP - add troubleshooting

### CODE REVIEW

**Comment by anfibiacreativa** [comment]
@kristenwomack @spboyer 
current blocker to complete this task:
- pushing a generated file with scan results to repos, must be investigated. It's likely we can't push not even to the existing PR branch and hence, we may need to find a different solution
https://github.com/Azure-Samples/template-pipeline-automation/issues/85
- we must agree that the security disclaimer added to the readme is mandatory for all templates
see https://github.com/Azure-Samples/azd-template-artifacts/pull/17/commits/f2bf48ed0a10f1730fedfae71301abdba80ac671
- the official docs page with links to security recommendations per service, must be in place
https://github.com/Azure-Samples/template-pipeline-automation/issues/84

File: README.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ Sample application code is included in this project. You can use or modify this
 
 (include a screenshot of your template's endpoint here-- so users know what it should look like when they're done)
 
+## Important Security Notice (Template Owners, do not remove!)
+
+This template, the application code and configuration it contains, has been built to showcase Microsoft Azure specific services and tools. We strongly advise our customers not to make this code part of their production environments without implementing or enabling additional security features.  
+
+Find recommendations for this specific template [in this generated document](./security-scan-results).
```
Comment (kristenwomack) [inline README.md]:
```suggestion

```

**Review by kristenwomack — APPROVED** [review]
Left one suggestion.

**Review by kristenwomack — APPROVED** [review]
LGTM

**Review by kristenwomack — COMMENTED** [review]
LGTM

**Pull request merged**